draft/accessibility-team.rst

```diff
@@ -0,0 +1,129 @@
+============================
+DEP XXXX: Accessibility Team
+============================
+
+:DEP: XXXX
+:Author: Tom Carrick
+:Implementation Team: Tom Carrick, others to be determined
+:Shepherd: to be determined
+:Status: Draft
+:Type: Process
+:Created: 2020-06-29
+:Last-Modified: 2020-06-29
+
+.. contents:: Table of Contents
+   :depth: 3
+   :local:
+
+Abstract
+========
+
+This DEP proposes the formation of an Accessibility Team to encourage projects
+maintained by the Django Software Foundation to be accessible to as diverse a
+group as possible, particularly people with disabilities that make using the
+web more difficult.
```

Comment:
Would it be worthwhile to link to a resource on who those people are? For example https://www.w3.org/WAI/people-use-web/abilities-barriers/ or https://www.w3.org/WAI/fundamentals/accessibility-usability-inclusion/.

Reply:
👍  - will add this.